### PR TITLE
Dialog Box UI of ContactPostStaff and ContactOtherStaff activities changed

### PR DESCRIPTION
**Pull request by rohands**
screenshot_2016-03-02-01-17-42

### CODE REVIEW

File: app/src/main/res/layout/dialog_listitem.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+
+    >
+
+
+    <ImageView
+        android:id="@+id/dialog_img"
+        android:layout_width="55dp"
```
Comment (Buddhiprabha) [inline app/src/main/res/layout/dialog_listitem.xml]: Dimensions should not be hard coded. Please add all to dimens.xml.

**Comment by rohands** [comment]
@Buddhiprabha 
@sandarumk 
I have updated the dimens.xml file

**Comment by sandarumk** [comment]
@Buddhiprabha Please have a look at this and merge this too, if it is ready to be merged.

**Comment by sandarumk** [comment]
@rohands Can you solve the conflicts here too so I can merge.

**Comment by rohands** [comment]
@sandarumk @Buddhiprabha 
I have resolved the conflicts.

File: app/src/main/res/layout/dialog_listitem.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+
+
+    <TextView
+        android:id="@+id/dialog_txt"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textSize="@dimen/text_large"
+        android:layout_gravity="center"
+        android:textColor="#FFFFFF"
```
Comment (sandarumk) [inline app/src/main/res/layout/dialog_listitem.xml]: refactor using the color.xml

File: app/src/main/java/com/peacecorps/pcsa/reporting/CustomAdapter.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+import com.peacecorps.pcsa.R;
+
+/**
+ * Created by Rohan on 25-02-2016.
+ * This adapter is to initialise views of the customised dialog box.
+ */
+public class CustomAdapter extends BaseAdapter {
+
+    public static String[] caption = {"Voice Call","Send message"};
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/reporting/CustomAdapter.java]: Do not add hard coded strings. Use strings.xml

File: app/src/main/java/com/peacecorps/pcsa/reporting/ContactPostStaff.java [hunk 4]
```diff
@@ -72,21 +73,24 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                CustomAdapter.createDialog("Contact SSM via", ContactPostStaff.this).show();
             }
         });
 
         contactSarl.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                createDialog(v, selectedLocationDetails.getSarlContact());
+                numberToContact = selectedLocationDetails.getSarlContact();
+                CustomAdapter.createDialog("Contact SARL via", ContactPostStaff.this).show();
```
Comment (sandarumk) [inline app/src/main/java/com/peacecorps/pcsa/reporting/ContactPostStaff.java]: Avoid using hard coded string segments. use the string.xml

**Comment by rohands** [comment]
@sandarumk @Buddhiprabha 
Would this be fine?

@sandarumk Sorry for multiple commits.

File: app/src/main/res/layout/dialog_list.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/bg_btn_rounded_rectangle">
+
+    <ListView
+        android:id="@+id/dialog_listview"
+        android:layout_width="fill_parent"
+        android:layout_height="fill_parent"
+        android:dividerHeight="@dimen/dialog_divider_height"
+        android:divider="#FFFFFF"
```
Comment (sandarumk) [inline app/src/main/res/layout/dialog_list.xml]: color to the xml

**Comment by sandarumk** [comment]
Can you submit a fresh PR with all the issues solved. Fewer commits. Refactored code. Go through your code and find any refactoring like strings and colors should be done and fix them too.

**Comment by rohands** [comment]
Alright, please close this PR. Thank you

**Comment by sandarumk** [comment]
I will close this PR once your new PR is submitted.

**Pull request closed without merging**